### PR TITLE
Comment HTML for a card

### PR DESCRIPTION
**Pull request by mpolimen**

### CODE REVIEW

**Comment by mpolimen** [comment]
This will presumably be a partial inside card html that displays the comments and adds a separate form to post a new comment

File: src/views/partials/comment-html.handlebars [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<h2>Comments</h2>
+{{#each comments}}
+<h3>{{user.initials}}</h3>
+<p class="comment">{{comment}}</p>
+{{/each}}
+<form id="comment-form" action="/boards/card/comments/{{boardId}}/{{cardId}}" method="PUT">
+     <label>
```
Comment (bwormser) [inline src/views/partials/comment-html.handlebars:7]: Same thing, not sure if it needs a for attribute but that's how we did it in class

Comment (mpolimen) [reply src/views/partials/comment-html.handlebars:7]: He showed us both 👍

File: src/views/partials/comment-html.handlebars [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+        Comment
+        <input type="text" name="comment" class="comment-box"/>
+    </label>
+    <button type="submit">Send</button>
+</form>
```
Comment (roccopolimen) [inline src/views/partials/comment-html.handlebars:12]: add something like:
<input type="hidden" name="_method" value="put" />

File: src/views/partials/comment-html.handlebars [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<h2>Comments</h2>
+{{#each comments}}
+<h3>{{user.initials}}</h3>
+<p class="comment">{{comment}}</p>
+{{/each}}
+<form id="comment-form" action="/boards/card/comments/{{boardId}}/{{cardId}}" method="PUT">
```
Comment (roccopolimen) [inline src/views/partials/comment-html.handlebars:6]: method cannot be PUT. Change to POST

**Review by roccopolimen — COMMENTED** [review]
quick change for method header

**Pull request merged**